config/redirects

```diff
@@ -0,0 +1,4 @@
+define: base https://www.mongodb.com/docs/mongodb-vscode
+
```

Comment:
[minor nit] following a similar pattern as other docs, can we define prefix and use it in the `base` definition as well as the first rule?

ie. 
```
define: prefix docs/mongodb-vscode
define: base https://www.mongodb.com/${prefix}

raw: ${prefix}/require-modules -> ${base}/require-playgrounds/require-modules/

Reply:
Done! Thanks for your feedback!